Parking-Lot/src/com/practice/parkinglot/Ticket.java

```diff
@@ -0,0 +1,35 @@
+package com.practice.parkinglot;
+import java.time.*;
+
+
+public class Ticket {
+	private String ticketId;
+	private String registrationNumber;
+	private LocalDateTime checkIn;
+	private LocalDateTime checkOut;

```

Comment:
If this was not part of requirement then it could have been avoided to save time.